HW04.Rmd

```diff
@@ -69,7 +122,12 @@ Assume that:
 * Fall = October-December
 
 ```{r season}
+gun_deaths_by_season <- gun_deaths_data %>%
+    mutate(______________________ c("Jan", "Feb", "Mar") <- Winter, c("Apr", "May", "Jun") <- Spring, c("Jul", "Aug", "Sep") <- Summer, c("Oct", "Nov", "Dec") <- Fall ) %>%
+
+count()
 
+# not sure what to do here please help :')

```

Comment:
I feel like you're close to doing something interesting and much more efficient and elegant than what I did. I don't know how to help though. I replaced the months with the appropriate season using gsub and plotted by month, which I feel like it required more code than was really necessary to get the job done.